pkg/webhook/validator.go

```diff
@@ -490,8 +490,53 @@ func ociSignatureToPolicySignature(ctx context.Context, sigs []oci.Signature) []
 				},
 			})
 		} else {
-			// TODO(mattmoor): Is there anything we should encode for key-based?
-			ret = append(ret, PolicySignature{})
+			ret = append(ret, PolicySignature{
+				// TODO(mattmoor): Is there anything we should encode for key-based?
+			})
+		}
+	}
+	return ret
+}
+
+type attestation struct {
```

Comment:
It might be good to document why this intermediate is necessary for shuffling things back and forth, it's not obvious why it's necessary.

Reply:
added a comment